lib/Transforms/IPO/StructFieldCacheAnalysisImpl.h

```diff
@@ -580,6 +598,15 @@ class CloseProximityBuilder {
   /// Collapse FRG to establish CPG
   void buildCloseProximityRelations();
 
+  /// Get CP relations of two fields
+  const CloseProximityPairType *getCloseProximityPair(FieldNumType F1,
+                                                      FieldNumType F2) const {
+    if (F1 < F2)
```

Comment:
The input of the function could be two same fields, so I can't assert here. I intended to return the value in the CPT (which should be 0) if two fields are the same.